### DISCUSSION TITLE
Add a Directory to the site?

### DISCUSSION THREAD

**Discussion by dave-b-armstrong** [GetPublii/Publii#1431] [General] [answered]
I would like to run my Publii site as the main CMS for my system, but also upload a React app into a subdirectory to run a separate app under a different URL on the same domain.  (https://mysite.com/newdirectory would serve the index.html that runs the react app).

There doesn't seem to be a way to do that via File Manager, although I did see a comment in the old forums that it was on the roadmap. 

Is there another way to accomplish this for those of us who are comfortable making direct data updates in the Sqlite file or at the file system level?

**Comment by candidexmedia** [comment]
I'm curious: any reason why you're opting for a subdirectory, and not a subdomain?

**Reply by dave-b-armstrong** [reply]
Just to simplify hosting - The react app I have in mind has no back-end, and my site is currently on github pages. So if I can just run it from a directory, I get a simple free hosting mechanism.

**Comment by dziudek** [comment, marked as answer]
@dave-b-armstrong - you can still manually put directories under /input/root-files/ of your website - you just won't have ability to edit and navigate inside this directory using Files Manager, but it should be copied during deployment (if it won't conflict with other website catalogs generated during deployment)

**Reply by dave-b-armstrong** [reply]
Perfect - I'm glad it was such a simple answer - that worked as expected.
Thanks for the answer!
